src/wallet/wallet2.cpp

```diff
@@ -2289,15 +2287,15 @@ void wallet2::load(const std::string& wallet_, const std::string& password)
 
       std::stringstream iss;
       iss << cache_data;
-      boost::archive::binary_iarchive ar(iss);
+      cereal::PortableBinaryInputArchive ar(iss);
```

Comment:
Thank you so much for reviewing, that was helpful!
The revised version handles the old boost format as well.